app/controllers/StorageExplorerController.scala

```diff
@@ -164,25 +201,41 @@ class StorageExplorerController @Inject() (
 
     future.map {
       case Some( vertex ) =>
-        if ( vertex.types.contains( NamespaceAndName( "resource:bucket" ) ) )
+        if ( vertex.types.contains( NamespaceAndName( "resource:bucket" ) ) ) {
+          logger.debug( "Returning metadata for bucket with id " + id )
           Ok( Json.toJson( vertex )( PersistedVertexFormat ) )
-        else
-          NotAcceptable // to differentiate from not found
-      case None => NotFound
+        }
+
+        else {
+          logger.debug( "Node with id" + id + " ??? " )
```

Comment:
it would be indeed quite strange to end up here...

Reply:
This one I found in the explorer, it was there before I started on it. I decided to leave it in because I was not able to come up with a way it might happen, but I was unsure if I missed something.